lib/comparison.js

```diff
@@ -277,7 +277,7 @@ helpers.ifEven = function(num, options) {
  */
 
 helpers.ifNth = function(a, b, options) {
-  if (++b % a === 0) {
+  if (b % a === 0) {
```

Comment:
None of the other helpers in the file are doing any casting/checks at present, however if you have an example of a helper that implements type checking and appropriate error messages I'll be happy to add it in.